### DISCUSSION TITLE
I sound like a noob but... Upgrading NGINX Proxy Manager?

### DISCUSSION THREAD

**Discussion by NorthantsAndy** [tteck/Proxmox#2475] [Help Needed] [answered]
I'm using v2.9.19 and after the recent PVE upgrade to 8.1.4 everything CLI works (dynamic IP, pinging HOSTS internally and externally against NPM host names) but via a browser, the sites don't work: I get too long to reply.

How do I manually upgrade to the latest NGINX PM from the CLI?  I  have look on the usual search engines, even asked a GPT for support but go nowhere... 

I am a sysadmin, but feel like such a noob....

**Comment by DJKatastrof** [comment, marked as answer]
its on the webpage,
![image](https://github.com/tteck/Proxmox/assets/64254359/2fae02e2-0459-40dc-b82c-68acab4e6731)

**Comment by NorthantsAndy** [comment]
Oh for goodness sake *facepalm*

Cheers chap...
